src/css/base.scss

```diff
@@ -1885,6 +1918,10 @@ textarea {
     width: 100px;
 }
 
+.w-120-px {
```

Comment:
Can we make it consistent? Like in the bottom we are using w-180 directly. We don't event use many percentage on 120 | 180 type digits